### ISSUE TITLE
Simplify the “User Groups” section: display groups as a grid, not a tree

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#15148]
### Summary
In the current version, it’s almost impossible to understand where to edit a specific “User Group”, because the editing link is hidden in the context menu.
And when you click on a group, it’s not the group’s settings appearing (what you expect), but adding a user to the group.

![users_groups_1](https://user-images.githubusercontent.com/12523676/86515911-3cb84c80-be25-11ea-9237-5ef740395169.gif)

![users_groups_2](https://user-images.githubusercontent.com/12523676/86515912-3d50e300-be25-11ea-9bd1-4801618befa8.gif)

**I suggest making the display of "User Group" in the form of a grid instead of a tree.** And the user can be added to the group inside the group editing (this already works, see gif above).
This will be much clearer, especially with the new link on name and gear context menu.

It will look something like this:

![users_groups_3](https://user-images.githubusercontent.com/12523676/86516279-e4cf1500-be27-11ea-8714-7fd46e7d31bc.png)

It will be possible to display useful information in the "User Group" grid, such as users in a group, access to sections (contexts, groups, etc.)

### Why is it needed?
Simplification in UI / UX

### Related issue(s)/PR(s)
N/A

**Comment by sergant210** [comment]
In addition
1. Rename the "User groups & users" tab to "User groups".
2. Simplify the copy logic of Access Policies .

**Comment by SnowCreative** [comment]
I agree that it's confusing that way it's set up now.

**Comment by sebastian-marinescu** [comment]
A "Update User group" or "Update group settings" button above the already loaded grid would also make it much clearer.

**Comment by Ruslan-Aleev** [comment]
> A "Update User group" or "Update group settings" button above the already loaded grid would also make it much clearer.

I do not quite understand.
Now all actions are located in the gear menu on the right, so an additional button is not needed.

**Comment by sebastian-marinescu** [comment]
Your suggestion and UI proposal look very good, assuming someone would provide a PR for that. 
I'm just saying that it might also be possible to add another button, like this: 

![image](https://user-images.githubusercontent.com/3798871/87715105-21e2c200-c7ad-11ea-923b-dfd6bc3ac5a0.png)

**Comment by Ruslan-Aleev** [comment]
Now it's clearer :) Yes, we can do this, and it's easier.
But personally, I am in favor of rewriting the section completely.
p.s. Probably for branch 2.8 the button solution would be appropriate.

**Comment by JoshuaLuckers** [comment]
I'm ok with having a button in 3.0 as well. A rewrite would be nice but it's something that can go into 3.1 as well.

**Comment by Ruslan-Aleev** [comment]
I was also always confused by the name `User Groups`, isn't it more logical to call it `Access Groups`?

**Comment by ghost** [comment]
Not necessarily; there are User Groups and Resource Groups. Access Groups could easily get confused for either of those.

**Comment by Bruno17** [comment]
> I was also always confused by the name `User Groups`, isn't it more logical to call it `Access Groups`?

Sometimes we group Users for other reasons than access. So User Groups is more logical for me.

**Comment by Mark-H** [comment]
The button has been added in 2.8 but I'll leave the issue open if people want to discuss further how users & user groups is currently set up with the tree.

**Comment by Ibochkarev** [comment]
Improved in #15270 

@modxbot close

**Comment by Ibochkarev** [comment]
@modxbot open

**Comment by Ruslan-Aleev** [comment]
In the future, it makes sense to remove other trees as well, if possible. Trees (other than resources and menus) are controversial UX choices, see https://github.com/modxcms/revolution/issues/10518, https://github.com/modxcms/revolution/pull/15478

**Comment by smg6511** [comment]
This idea would be a no-brainer to me ... _except_ ... how would we handle the nesting of user groups (which is allowable and easy to show/manage in the tree view)? I haven't been called on by any of my modx clients to set up user groups where subcategorization would be beneficial; as such, the proposed simplification would not be a problem for me. For others it could be problematic.

Anyway, something to think about before someone dives into a PR at some point...
